Detectors/MUON/MCH/Simulation/src/Stepper.h

```diff
@@ -0,0 +1,55 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_MCH_SIMULATION_STEPPER_H
+#define O2_MCH_SIMULATION_STEPPER_H
+
+#include "MCHSimulation/Hit.h"
+#include "TVirtualMC.h"
+#include <iostream>
+#include <vector>
+#include <array>
+
+namespace o2
+{
+namespace mch
+{
+
+class Stepper
```

Comment:
what is the purpose of this class?

Reply:
ok fine. You probably delegate the `Detector::ProcessHits` function to this guy.